### PR TITLE
Updated Deleting application

### PR DESCRIPTION
**Pull request by gtrivedi88**
Purpose: The purpose of this PR is to review and update Deleting application and component content. I have copied and updated the AppStudio content. Next tasks is to review and update it.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/deleting.adoc [hunk 1]
```diff
@@ -1 +1,41 @@
-= Deleting applications and components
\ No newline at end of file
+= Deleting applications and components
+
+You can delete an application if you are the owner or if you have admin access to the application.
```
Comment (arewm) [inline docs/modules/ROOT/pages/how-tos/deleting.adoc:3]: I would not say owner or admin access. Instead, you can say something like if you are a user with the workspace admin role.

File: docs/modules/ROOT/pages/how-tos/deleting.adoc [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+WARNING: If you delete an application permanently, you delete all the components and resources from the user interface (UI) and upstream repositories. You cannot undo the deletion.
```
Comment (arewm) [inline docs/modules/ROOT/pages/how-tos/deleting.adoc:5]: What resources are contained in upstream repositories? I guess that the deletion submits a PR to delete the tekton pipleine definitions. Is that what you were referring to?

File: docs/modules/ROOT/pages/how-tos/deleting.adoc [hunk 3]
```diff
@@ -1 +1,41 @@
...
+
+== Deleting an application in the web UI
+.*Prerequisites*
+
+* You have successfully signed into {ProductName}.
+* You have at least one application.
```
Comment (arewm) [inline docs/modules/ROOT/pages/how-tos/deleting.adoc:11]: These feel like no-op prerequisites. A better one might just be having the `admin` role.

File: docs/modules/ROOT/pages/how-tos/deleting.adoc [hunk 4]
```diff
@@ -1 +1,41 @@
...
+
+.*Procedures*
+
+. In the console, select *My applications*.
```
Comment (arewm) [inline docs/modules/ROOT/pages/how-tos/deleting.adoc:15]: Are we calling it a console or a UI?

File: docs/modules/ROOT/pages/how-tos/deleting.adoc [hunk 5]
```diff
@@ -1 +1,41 @@
...
++
+NOTE: You can also delete an application by selecting *Actions > Delete Application*.
+
+
+== Deleting an application using CLI
+
+.*Prerequisites*
+
+* You have successfully signed into {ProductName}.
+* You have at least one application.
```
Comment (arewm) [inline docs/modules/ROOT/pages/how-tos/deleting.adoc:28]: Similar content to above, but maybe also that you have configured the CLI?

**Comment by gtrivedi88** [comment]
@arewm Addressed all comments. Could you please review and me know  your approval or suggestions?

**Pull request merged**